system/integrations/pgp/index.rst

```diff
@@ -0,0 +1,111 @@
+   That means you have to choose a group with a sender email account, which has a private key assigned.
+
+Handling of keys
+-----------------
+To use the pgp function, you fist have to enable the integration (PGP). 
+When you swith the toggle to *enabled*, you can add keys by clicking the *add key* button. 
+The keys can be imported from a file or you can paste the content of the key in the text box.
+
+.. note:: **Which keys do I have to import?**
+
+   For **signing** *outgoing* emails, you have to import the private key of your sender email account.
+   
+   For **encrypting** *outgoing* emails, you have to import the public key of the recepient's email account.
+
+   For **checking the signature** of signed *incoming* emails, you have to import the public key of the sender.
+
+   For **decrypting** of encrypted *incoming* emails, you have to import the public key of the sender. 
+
+Import keys from a file
+`````````````````````````
+You can import keys from a file in the section **Upload key**:
+
+.. figure:: /images/system/integrations/pgp/import_key.png
+   :alt: Screenshot of adding a key via file
+   :align: center
+
+Import keys by pasting the content
+`````````````````````````````````````
+You can also paste the key's content in the section **paste key**:
+
+.. figure:: /images/system/integrations/pgp/paste_key.png
+   :alt: Screenshot of adding a key via pasting
+   :align: center
+
+
+Deleting keys
+```````````````
+If you want to delete a specific key, you can do it by clicking on the menu in the *actions* column and select **delete**:
+
+.. figure:: /images/system/integrations/pgp/delete_key.png
+   :alt: Screenshot of deleting a key
+   :align: center
+
+
+Downloading keys
+`````````````````
+If you want to download your keys, you can do this as well in the corresponding actions button. 
+Depending on the key, you can choose wether you want to download the private or the public key.
+
+
+Default behavior
+-----------------
+Here you can adjust on per group basis, if *sign* and *encryption* is on or off by default. 
+Please be aware, that agents can always override the setting for each individual article. 
+
+.. figure:: /images/system/integrations/pgp/group_default.png
+   :alt: Adjusting default behavior on per group basis
+   :align: center
+
+
+Recent logs
+-------------
+Here you can see the last actions regarding signing and encryption and if they were succesfull. 
+
+
+Troubleshooting
+----------------
+
+**Sign button is not visible, but keys are imported.**
+   * Did you choose a group in the ticket?
+   * Did you import a private key for the email adress, which is used for outgoing emails in the group?
+
+
+**How to obtain keys?**
+   You can create them yourself! Please have a look :doc:`here </system/integrations/pgp/generate_keys>` for details.
+
+**It says a passphrase is needed, but I haven't one.**
+   If the key is secured with a passphrase, you have to provide it for the import in Zammad. 
```

Comment:
Please mention that a key may have an empty passphrase, although it's not recommended.